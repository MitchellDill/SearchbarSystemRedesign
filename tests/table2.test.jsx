import React from "react";
import renderer from "react-test-renderer";
import Table2 from "../client/table2";

it.skip("continues to render Table2 component correctly", () => {
  const tree = renderer.create(<Table2 />).toJSON();
  expect(tree).toMatchSnapshot();
});
