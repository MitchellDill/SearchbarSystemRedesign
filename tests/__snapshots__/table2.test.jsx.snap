// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`continues to render Table2 component correctly 1`] = `
<div
  className="T2"
>
  <hr />
  <div
    className="table2"
  >
    <table
      className="nav"
    >
      <tbody>
        <tr>
          <td>
            Home
          </td>
          <td>
            Saved
          </td>
          <td>
            Motors
          </td>
          <td>
            Fashion
          </td>
          <td>
            Electronics
          </td>
          <td>
            Collectibles & Art
          </td>
          <td>
            Home & Garden
          </td>
          <td>
            Sporting Goods
          </td>
          <td>
            Toys
          </td>
          <td>
            Business & industrial
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
